### PR TITLE
Ignore stores to the return value of the interp_step.

### PR DESCRIPTION
**Pull request by vext01**

### CODE REVIEW

File: yktrace/src/tir.rs [hunk 1]
```diff
@@ -179,6 +181,20 @@ impl<'a> TirTrace<'a> {
                         // The following statements are specific to TIR and cannot appear in SIR.
                         Statement::Call(..) | Statement::StorageDead(_) => unreachable!()
                     };
+
+                    // In TIR, stores to local number zero are always to the return value of the
+                    // #[interp_step] function. We know this is unit so we can ignore it.
```
Comment (bjorn3) [inline yktrace/src/tir.rs:186]: Maybe an assert that the return type is unit?

Comment (vext01) [reply yktrace/src/tir.rs:186]: Fixed in 6ce00f5, thanks.

**Comment by ptersilie** [comment]
Please squash.

**Comment by vext01** [comment]
splat.

**Pull request merged**